client/src/components/UpcomingEventsCard/UpcomingEventsCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./UpcomingEventsCard.css";
+// import { prependOnceListener } from 'cluster';
+import { Link } from "react-router-dom";
+
+const UpcomingEventsCard = (props) => {
+
+  return(
+    <Link className="UpcomingEventsCard" to="#">
+      <span className="UpcomingEventsCard__name">{props.name}</span>
+      <span className="UpcomingEventsCard__date">Date: {props.date}</span>
+      <span className="UpcomingEventsCard__time">Time: {props.time}</span>
+      <span className="UpcomingEventsCard__location">Location: {props.location}</span>
+    </Link>
```

Comment:
Try importing the `Info Card, Pink, Small` card here and wrapping the link around that. From there, you can add the name, date, time and location properties. The `Info Card, Pink, Small` should have a min-width and other CSS properties that your `UpcomningEventsCard` can inherit so that your location isn't running off the bottom edge of the card.

Reply:
Is there a way to import just the `Info Card, Pink, Small`? Or do I import all of the Card parent component and just add the `color` and `size` attributes in UpcomingEventsCard.js? I'm not quite sure how to go about this.